6.bulk_Morphology_Elastic_Network/0.split_data/notebooks/0.split_data_regression.ipynb

```diff
@@ -1,872 +1,912 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    test_set_50 = data_df[
</pre><p>It looks like you are assigning a different value to this variable immediately after assigning a value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>fixed this error</p>